src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -128,6 +128,11 @@ public TagHelperContent AppendFormat(IFormatProvider provider, string format, pa
         /// <returns>A reference to this instance after the clear operation has completed.</returns>
         public abstract TagHelperContent Clear();
 
+        /// <summary>
+        /// Clears the <see cref="TagHelperContent"/> so it can be reused.
+        /// </summary>
+        public abstract void Reset();
```

Comment:
Add `<remarks>` distinguishing this method from `Clear()`. This question will come up repeatedly otherwise, especially given the word "Clears" in the `<summary>`.